experimental/dds/tree2/src/test/tree/anchorSet.spec.ts

```diff
@@ -245,37 +245,32 @@ describe("AnchorSet", () => {
 
 		node0.on("childrenChanging", log.logger("childrenChange"));
 		node0.on("subtreeChanging", log.logger("subtreeChange"));
-		node0.on("valueChanging", log.logger("valueChange"));
 		node0.on("afterDelete", log.logger("afterDelete"));
 
 		log.expect([]);
 
-		const valueMark: Delta.Modify = {
-			type: Delta.MarkType.Modify,
-			setValue: "x",
+		const insertMark: Delta.Insert = {
+			type: Delta.MarkType.Insert,
+			content: [singleTextCursor({ type: jsonString.name, value: "x" })],
 		};
-		anchors.applyDelta(new Map([[rootFieldKeySymbol, [0, valueMark]]]));
+		anchors.applyDelta(new Map([[rootFieldKeySymbol, [0, deleteMark, 0, insertMark]]]));
```

Comment:
In the delta format, numbers represent offsets so `0` does nothing.
```suggestion
		anchors.applyDelta(new Map([[rootFieldKeySymbol, [deleteMark, insertMark]]]));
```